lib/temp-directory.js

```diff
@@ -21,7 +22,17 @@ function create(context, next) {
     if (err) {
       return next(err);
     }
-    return next(null, context);
+    /* Tells eslint running in module tests not to look for config files above
+       this temp directory. */
+    const eslintrcPath = path.join(context.path, '.eslintrc.yml');
```

Comment:
it might be nice to break this out into it's own helper function to avoid the code floating right